schema/crdb/dbinit.sql

```diff
@@ -639,7 +639,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.region (
 
     read_only BOOL NOT NULL,
 
-    deleting BOOL NOT NULL
+    deleting BOOL NOT NULL,
+
+    reservation_factor FLOAT NOT NULL
```

Comment:
It seems like we might want a `CHECK reservation_factor >= 1.0` here.

Reply:
this changed to an enum, the value is no longer unrestricted